src/diffpy/utils/tools.py

```diff
@@ -71,67 +80,64 @@ def load_config(file_path):
 def _sorted_merge(*dicts):
     merged = {}
     for d in dicts:
+        d = _clean_dict(d)
         merged.update(d)
     return merged
 
 
-def _create_global_config(args):
+def _create_global_config(user_info):
     username = input(
-        f"Please enter the name you would want future work to be credited to " f"[{args.get('username', '')}]:  "
-    ).strip() or args.get("username", "")
-    email = input(f"Please enter the your email " f"[{args.get('email', '')}]:  ").strip() or args.get("email", "")
-    return_bool = False if username is None or email is None else True
-    with open(Path().home() / "diffpyconfig.json", "w") as f:
-        f.write(json.dumps({"username": stringify(username), "email": stringify(email)}))
-    print(
-        f"You can manually edit the config file at {Path().home() / 'diffpyconfig.json'} using any text editor.\n"
-        f"Or you can update the config file by passing new values to get_user_info(), "
-        f"see examples here: https://www.diffpy.org/diffpy.utils/examples/toolsexample.html"
+        f"Please enter the name you would want future work to be credited to "
+        f"[{user_info.get('username', '')}]:  "
+    ).strip() or user_info.get("username", "")
+    email = input(f"Please enter the your email " f"[{user_info.get('email', '')}]:  ").strip() or user_info.get(
+        "email", ""
     )
-    return return_bool
+    config = {"username": _stringify(username), "email": _stringify(email)}
+    if username and email:
```

Comment:
The function originally returns a bool that passes back to `get_user_info` to see if we need to create a global config file. If user only input none or one of username / email, an empty global config file will be created. I simplified the logic here so that we only create a global config file if both username and email are present. Does this make sense though?